### DISCUSSION TITLE
flet build apk error

### DISCUSSION THREAD

**Discussion by ChinaGodzilla** [flet-dev/flet#3530] [General]
![image](https://github.com/flet-dev/flet/assets/43966525/8aa7ab8a-53c4-4b29-b340-d94c28c344b0)

The APK could be compiled successfully before, but this problem occurred after upgrading flet to the latest version. After that, many related supporting components such as flutter, git, etc. were upgraded, but it still didn't work.

Thanks for your time!

**Comment by FeodorFitsner** [comment]
Are you using Flet from sources or what?

**Comment by FeodorFitsner** [comment]
I've just published 0.24.0 branch of template anyway.

**Reply by ChinaGodzilla** [reply]
![image](https://github.com/flet-dev/flet/assets/43966525/6e8d95d1-e66a-4750-9b98-24c15cf5b3b4)
Sorry for forgetting to provide the version, thank you

**Reply by FeodorFitsner** [reply]
OK, try now.

**Reply by ChinaGodzilla** [reply]
@FeodorFitsner 

![image](https://github.com/flet-dev/flet/assets/43966525/3c2065bc-dbba-47da-87ca-b0f293dfb2dd)


After downgrading, the error did not appear

**Reply by burhansvural** [reply]
> OK, try now.

Flet application on Google Play on Android API 34-35 also gives an error and does not work.

**Reply by FeodorFitsner** [reply]
Build with Flet 0.23.1. What error do you get?

**Comment by ChinaGodzilla** [comment]
![image](https://github.com/flet-dev/flet/assets/43966525/07eefb23-8ea3-46f1-8292-12894e3c8bdc)
No environment changes were made, only the flet version was changed, and the APK was built successfully
